manifests/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -205,8 +205,15 @@ template: |
             {{ $container.Name }}
           {{- end}}
         ]
+    {{ $deprecatedClusterID := .Values.global.multiCluster.clusterName }}
```

Comment:
I'm not sure deprecating clusterName has been properly discussed/approved or is worth it.